main.go

```diff
@@ -0,0 +1,127 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"strconv"
+	"time"
+
+	"github.com/aws/aws-lambda-go/lambda"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/s3/s3manager"
+	"github.com/aws/aws-sdk-go/service/ssm"
+)
+
+func main() {
+	lambda.Start(HandleLambdaEvent)
+}
+
+func HandleLambdaEvent() {
+	var r RealRunner
+	runReport(r)
```

Comment:
```suggestion
	runReport(&RealRunner{})
```
We pass struct directly without variable if we are not reusing it.